tests/acceptance/05_processes/01_matching/select_without_result.cf

```diff
@@ -0,0 +1,86 @@
+#######################################################
+#
+# Check process_select bodies without process_result attribute.
+#
+# If process_select bodies do not have a process_result attribute, its
+# given attributes should be AND'ed together for process filtering.
+#
+# Tests:
+#
+# 1.  Negative: using a highly unlikely virtual memory size range.
+#
+# 2.  Positive: using a virtual memory size range, that should match
+#               all processes
+#
+#######################################################
+
+body common control
+{
+      inputs => { "../../default.cf.sub" };
+      bundlesequence  => { default("$(this.promise_filename)") };
+      version => "1.0";
+}
+
+#######################################################
+
+bundle agent init
+{
+  vars:
+      "dummy" string => "dummy";
+}
+
+#######################################################
+
+bundle agent test
+{
+  meta:
+      "description" -> {"CFE-4511"}
```

Comment:
We need to look into a failure in windows for this test. We get a bunch of:

```
2025-04-04T19:37:46Coordinated Universal Time    error: Anomalous ending ".5" while parsing integer number: 0.5
```

Reply:
Looking at the docs https://docs.cfengine.com/docs/3.24/reference-promise-types-processes.html#vsize it would seem vsize should work on Windows:

> On Windows, the virtual memory size is the amount of memory that cannot be shared with other processes. In Task Manager, this is called Commit Size (Windows 2008), or VM Size (Windows XP).

Reply:
"The failed to extract a valid integer from" with emptiness after it is related to the ppid. I only see that when I comment out the vsize body attribute.

Reply:
I will check if this is a regression and if so we can just mark this test as "needs work" on windows and give a ticket number.

Reply:
Yeah, this is an existing issue in 3.25.0 apparently. It seems the vsize values in Windows are real type and are being attempted to be parsed as integers.

I will add the meta promise needed now for a new ticket I created for the issue: ENT-12751

Reply:
Ah, I can't push I think due to lack of rights on your repo. Makes sense.

Please include this change either as a separate commit or squashed into the one so far.

```suggestion
      # This test exposes a known issue with processes on the Windows platform.
      "test_soft_fail" -> {"ENT-12751"}
        string => "windows";

      "description" -> {"CFE-4511"}
```